### PR TITLE
Use paper's 1.21.4 API from spigot's 1.21.1

### PR DESCRIPTION
**Pull request by 29cmb**
Due to Paper becoming a hard fork of Spigot, and the servers all using it, using a spigot plugin would eventually break the plugin on the server. So I decided to go over to the Paper API, but this came with the deprecation of a lot of stuff with it nudging us towards the Adventure components API, being built in.

RIP ChatColor, you will be missed

gl @Nibbl-z, you have to review this now

### CODE REVIEW

**Comment by Nibbl-z** [comment]
noooooooooooooooooo 😨

File: src/main/java/xyz/devcmb/cmr/commands/cosmetics/RollCrateCommand.java
Comment (Nibbl-z) [inline src/main/java/xyz/devcmb/cmr/commands/cosmetics/RollCrateCommand.java:1]: bring back rollcratecommand.java 🙏

File: src/main/java/xyz/devcmb/cmr/commands/game/MinigameCommand.java [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
-    }
-
     @Override
     public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command command, @NotNull String s, @NotNull String[] args) {
         if(args.length == 1){
             Minigame minigame = GameManager.getMinigameById(args[0]);
             if(minigame != null){
-                // don't worry nibbles, i don't understand this enough either
-                audiences.sender(commandSender).sendMessage(Component.text()
+                commandSender.sendMessage(Component.text()
```
Comment (Nibbl-z) [inline src/main/java/xyz/devcmb/cmr/commands/game/MinigameCommand.java:24]: he underst ands

File: src/main/java/xyz/devcmb/cmr/utils/MapLoader.java [hunk 2]
```diff
@@ -65,6 +65,7 @@ public static void unloadMap() {
             MVWorldManager worldManager = multiverseCore.getMVWorldManager();
             if(worldManager.isMVWorld(LOADED_MAP)){
                 worldManager.deleteWorld(LOADED_MAP);
+                LOADED_MAP = null;
```
Comment (Nibbl-z) [inline src/main/java/xyz/devcmb/cmr/utils/MapLoader.java:68]: please briung back loaded map

**Review by Nibbl-z — APPROVED** [review]
bring back chatcolor fr

**Comment by Nibbl-z** [comment]
aaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by Nibbl-z** [comment]
hey guys did you know devcmb gets an email notifcation every single time i add a comment to this pr which might negatively affect his mcsr performance

**Comment by 29cmb** [comment]
> hey guys did you know devcmb gets an email notifcation every single time i add a comment to this pr which might negatively affect his mcsr performance

Jokes on you! I got a 22:42 hahahhahahahahha

**Pull request merged**